README.md

```diff
@@ -133,6 +133,24 @@ Run your app in production mode:
 npm run prod
 ```
 
+Run your app with an attached HTML Validator (HTML Validator is attached by default):
+
+```
+npm run dev attach
+```
+
+Run your app with a detached HTML Validator:
+
+```
+npm run dev detach
+```
+
+After running your app with a detached HTML Validator, use this command to shut down the HTML Validator:
+
+```
+npm run killValidator
+```
```

Comment:
Add this here:

*Note: if your Roosevelt app was created with an older version, you may not have this command in your npm scripts. If so, then manually add this npm script to your package.json: `"killValidator": "node ./node_modules/roosevelt/lib/killValidator.js"`*